nodeschool/amsterdam
# Update slides for NodeSchool 23

### What has been done?
Updated the slides for the next meetup

- Mentors
- Location/sponsor
- Workshops
  - excluded the ones I didn't test (Node LTS on MacOS + Windows) from the `Core workshoppers` and `Recommended electives` slides
  - the workshops in the `💀 electives` have not been tested, but I suspect people doing the most challenging workshops will have less issues running into possible Node compatibility issues or challenges not working. We can also do some expectation management by telling in the introduction that these workshoppers might be outdated
    - replaced `goingnative` with `shader-school`, because `goingnative` doesn't seem to work on LTS, and I believe shaders are just as challenging. Or maybe even more!

### How to test / run the presentation

- `nvm install 4 && nvm use 4`
- `npm i`
- `npm run start`

You should see something like this in your terminal and the slides in your browser on the given adresss:
```sh
Running "connect:server" (connect) task
Started connect web server on http://localhost:8000
```

### Notes
- I coudn't get the presentation to run with the LTS or other newer Node versions and didn't have the time to fix this, so let's just use Node 4. It works 😌
- Just edit/change whatever you like! 😊
